### PR TITLE
Gpt2 notebook example

### LINKED ISSUE #3216: GPT2-Triton Example: extand to contain load test example
extend current notebook to contain an example on load test using Vegeta tool

### PR DESCRIPTION
**Pull request by nadinet**
**What this PR does / why we need it**:
Extend notebook to contain an example of a simple load test using Vegeta 
**Which issue(s) this PR fixes**:

Fixes #3216 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
```
NONE

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **axsaucedo**
You can assign the PR to them by writing `/assign @axsaucedo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: examples/triton_gpt2/README.ipynb [hunk 1]
```diff
@@ -9,17 +9,20 @@
     "\n",
```
Comment (adriangonz) [inline examples/triton_gpt2/README.ipynb:9]: <p>Nice one on adding the <code>default</code>  namespace explicitly </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/3217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (adriangonz) [inline examples/triton_gpt2/README.ipynb:9]: <p>Should we remove the <code>try</code>  / <code>except</code>  block? I.e. just leave the <code>res = ret.json()</code>  to simplify the snippet?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/3217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (adriangonz) [inline examples/triton_gpt2/README.ipynb:9]: <p>Would it make sense to run <code>black</code>  over this snippet and the previous one? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/3217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[axsaucedo](https://github.com/SeldonIO/seldon-core/pull/3217#issuecomment-846125030 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Failed to merge this PR due to:
>failed merging [3217]: [Method Not Allowed]

File: examples/triton_gpt2/README.ipynb (hunk 1 above)
Comment (nadinet) [reply examples/triton_gpt2/README.ipynb:9]: <p>I added this block as when running in loop (while) may cause some request  to return with empty ret =&gt; ret.json() throw exception. I believe it is a timing issue/ load on the model as without calling request in loop never happens to me.  I added the try block to make sure the script returns a sentences completion. </p><p> </p><p>I find it O.K to leave it that way - I don't think it impairs the readability / understanding. Let me know if you prefer it to be removed </p>

Comment (nadinet) [reply examples/triton_gpt2/README.ipynb:9]: <p>It makes sense - ran it</p><p> </p>

**Pull request merged**